src/plugins/streaming-plugin.js

```diff
@@ -428,15 +428,21 @@ class StreamingHandle extends Handle {
    * @param {string} [params.secret]
    * @returns {Promise<module:streaming-plugin~STREAMING_EVENT_OK>}
    */
-  async startRecording({ id, audio, video, data, secret }) {
+  async startRecording({ id, audio, video, data, media, secret }) {
```

Comment:
- jsdocs: use a generic optional array of objects and start property description with `[multistream] Blah Blah`.